industrial_ci/src/tests/abi_check.sh

```diff
@@ -187,7 +187,11 @@ function run_abi_check() {
         export PATH="/usr/lib/ccache:$PATH"
     fi
 
-    extend="/opt/ros/$ROS_DISTRO"
+    if [ -n "$EXTEND_FROM" ]; then
```

Comment:
this decision should be made in `run.sh`. And then just used in all of these places.

Reply:
```
if [ -n "$ROS_DISTRO:-" ]; then
    EXTEND_FROM=${EXTEND_FROM:-/opt/ros/$ROS_DISTRO}
fi
```
in run.sh

and then
```
extend=${EXTEND_FROM:?}
```

Reply:
done